packages/grafana-data/src/types/logs.ts

```diff
@@ -33,6 +33,11 @@ export enum LogsMetaKind {
   LabelsMap,
 }
 
+export enum LogsSortOrder {
```

Comment:
I have renamed `SortOrder` to `LogsSortOrder` for more semantic and clear name and moved it to `grafana/data` so we can use it in `grafana/ui` components such as `LogsRow`.